ibc-primitives/src/types/timestamp.rs

```diff
@@ -309,6 +311,18 @@ impl std::error::Error for TimestampError {
     }
 }
 
+#[cfg(feature = "arbitrary")]
+fn arb_tendermint_time(
+    u: &mut arbitrary::Unstructured<'_>,
+) -> arbitrary::Result<PrimitiveDateTime> {
+    let secs: u64 = u.int_in_range(0..=u64::MAX)?;
```

Comment:
is there a reason that you didn't use

```rs
let secs: u64 = arbitrary::Arbitrary::arbitrary(u)?;
```

Reply:
I switched to use this the simpler form in [80f89a1](https://github.com/cosmos/ibc-rs/pull/1390/commits/80f89a1caa998a9da88838b392e7cd38584472a9)